app/src/pages/ui/OptionsSelector.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import ArrowDropUp from "@material-ui/icons/ArrowDropUp";
+import ArrowDropDown from "@material-ui/icons/ArrowDropDown";
+import TextField from "@material-ui/core/TextField";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import { Fips } from "../../utils/madlib/Fips";
+import Popover from "@material-ui/core/Popover";
+import Button from "@material-ui/core/Button";
+import styles from "./OptionsSelector.module.scss";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+
+function OptionsSelector(props: {
```

Comment:
In the future, if you're making a new file that's very similar to an old file that's deleted, you can use `git mv` to make sure the diff shows up against the old file

Reply:
oh ok! good to know!